README.md

```diff
@@ -26,13 +26,22 @@ First you'll need to make sure you have a full development environment set up:
 #### (macOS)
 
 ```
+brew update
 brew install cmake openssl tor ncurses
```

Comment:
testing now, macos `readlink` is BSD version, so add coreutils and change to `greadlink` 

`brew install cmake openssl tor ncurses coreutils`